src/Components/main.js

```diff
@@ -6,7 +6,7 @@ import Typography from '@material-ui/core/Typography';
 // Dependencies created by our group
 import { mainStyles } from '../styles/styles';
 import { HourGraph } from './Graphs/Hourgraph';
-import { ForecastGraph } from './Graphs/ForecastGraph';
+import { ForecastGraph } from './Graphs/Forecastgraph';
```

Comment:
Since 'Forecast' and 'Graph' are two words, I think we might want to use the convention of 'ForecastGraph'.

Reply:
Great idea. I just changed the names.